client/signup/steps/page-picker/index.tsx

```diff
@@ -205,24 +208,43 @@ const StyledButton = styled( Button )`
 `;
 
 export default function DIFMPagePicker( props: StepProps ) {
+	const { stepName, goToNextStep } = props;
+
 	const translate = useTranslate();
+	const dispatch = useDispatch();
+	const [ selectedPages, setSelectedPages ] = useState< string[] >( [
+		HOME_PAGE,
+		ABOUT_PAGE,
+		CONTACT_PAGE,
+	] );
 
 	const headerText = translate( 'Add pages to your {{wbr}}{{/wbr}}website', {
 		components: { wbr: <wbr /> },
 	} );
 	const subHeaderText = translate( 'You can add up to 5 pages' );
+
```

Comment:
Please also add this snippet here so that the step is saved in the signup progress store:
```
useEffect( () => {
		dispatch( saveSignupStep( { stepName } ) );
}, [ ] );
```